src/Config.hs

```diff
@@ -58,21 +73,31 @@ data Environment
     deriving (Eq, Show, Read)
 
 -- | This returns a 'Middleware' based on the environment that we're in.
-setLogger :: Environment -> Middleware
-setLogger Test = id
-setLogger Development = logStdoutDev
-setLogger Production = logStdout
+setLogger :: Environment -> LogEnv -> Middleware
+setLogger Test _          = id
+setLogger Development env = katipLogger env
+setLogger Production env  = katipLogger env
+
+-- | Web request logger (currently unimplemented). For inspiration see
+-- ApacheLogger from wai-logger package.
+katipLogger :: LogEnv -> Middleware
+katipLogger env app req respond = runKatipT env $ do
+    -- todo: log proper request data
+    logMsg "web" InfoS "todo: received some request"
+    liftIO $ app req respond
```

Comment:
I'm inclined to say that it should be fine to continue using the original request middleware loggers. The request middleware logging is distinct from the more structured app logging that we might want to do.

Reply:
I would be happy to see a further PR that implemented the request middleware logging in katip; though that may even be worthy of it's own library :smile: